trino/auth.py

```diff
@@ -33,7 +33,7 @@
 from requests.auth import extract_cookies_to_jar
 
 import trino.logging
-from trino.client import exceptions
+from trino import exceptions
```

Comment:
Without this change:
```
ImportError while importing test module 'trino-python-client/tests/unit/test_types.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../.cache/pyenv/versions/3.9.21/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/unit/test_types.py:19: in <module>
    from trino import types
trino/__init__.py:12: in <module>
    from . import auth
trino/auth.py:36: in <module>
    from trino.client import exceptions
trino/client.py:67: in <module>
    from trino.auth import Authentication
E   ImportError: cannot import name 'Authentication' from partially initialized module 'trino.auth' (most likely due to a circular import) (trino-python-client/trino/auth.py)
```
Makes sense but not sure why `trino.client` was referred for `exceptions` in the first place.